spec/models/user_spec.rb

```diff
@@ -1,13 +1,11 @@
-require "spec_helper"
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.